dashdotdb/models/dashdotdb.py

```diff
@@ -121,8 +119,9 @@ class Vulnerability(db.Model):
     __tablename__ = 'vulnerability'
 
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(64), unique=False, index=True)
+    name = db.Column(db.String(10000), unique=False, index=True)
```

Comment:
Can we make this a little bit tighter? I can see things like descriptions to have 10k characters, but looks weird for a name.

Reply:
I also updated [imagemanifestvuln.json](examples/imagemanifestvuln.json) with an up-to-date format and I ran into errors due to column too small issues. Apparently the vulnerability name can now be more than 64 chars, e.g.,

```
RHSA-2022:7720: e2fsprogs security and bug fix update (Moderate)
```

This is not really a proper name anylonger, but more of a description in disguise, which made me worry on how long this could be in the end.
My thought here was that we actually could also delete old entries and thus dont really have to worry about space issues. As it stands, we only use the most recent entries in our reports.

Then again I can also see your point that this might be overkill. Will reduce to 512 instead. I guess that should be fine too.